change/@fluentui-react-tag-picker-964ccddd-d59e-4f97-8b75-992ddeb4681a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32297&commitId=bc11d718cc8d9b5dedc89fe7eedb71489cc7a548&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|15| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|1141| Changed|

</details>


  <details><summary><b>TagPicker</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Disabled Dark Mode.chromium.png|393| Changed|
|TagPicker.Disabled High Contrast.chromium.png|645| Changed|
|TagPicker.Disabled RTL.chromium.png|378| Changed|
|TagPicker.disabled.chromium.png|378| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>